hexa/workspaces/models.py

```diff
@@ -68,7 +68,7 @@ def create_if_has_perm(
             )
 
         db_password = User.objects.make_random_password(length=16)
-        db_name = format_db_name(create_kwargs["slug"])
+        db_name = format_db_name(str(uuid.uuid4()))
```

Comment:
We don't need `format_db_name` anymore - let's use `str(uuid.uuid4()).replace("-", "")`

Reply:
Okay